bazel/external/libcircllhist.BUILD

```diff
@@ -0,0 +1,17 @@
+cc_library(
+    name = "libcircllhist",
+    srcs = ["src/circllhist.c"],
+    hdrs = [
+        "src/circllhist.h",
+        "src/circllhist_config.h",  # Generated.
+    ],
+    includes = ["src"],
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "circllhist_config",
+    srcs = ["src/circllhist_config.h.in"],
```

Comment:
Can we follow up on @jmillikin-stripe comment here? It does seem weird that the generated header is just being copied. Can you add a comment as to why this is OK if it is? cc @postwait

Reply:
The _config.h.in is an autoconf input template.  It is not safe to copy it on all platforms.  There is a
```
#ifdef HAVE_ALLOCA_H
```

in the circllhist.c file that needs that that conditionally set depending on system headers.

Reply:
@postwait  thanks. What is your suggestion here? Should I copy based on platform? or which platforms it is good to copy?

Reply:
libcircllhist (like many open source projects) is auto-tools based.  When you build it, it will build-time detect which features are available on your platform and act on that.  I don't know how to do the same thing in cmake.  I am pretty sure that HAVE_ALLOCA_H is the only important thing in that header though -- so if you can get cmake to provide that, you're set.

Reply:
Looking at this, it is only alloca() that is a problem.  I've opened a PR on libcircllhist to resolve this.  It should be possible shortly to just ignore that header completely.

Reply:
@postwait thank you so much.

Reply:
@postwait Awesome. Thanks. I got the new version and removed the genRule and it works.